analyzer/language/golang/binary/binary.go

```diff
@@ -36,6 +36,11 @@ func (a gobinaryLibraryAnalyzer) Required(_ string, fileInfo os.FileInfo) bool {
 		return false
 	}
 
+	const MaxFileSize = 200000000
```

Comment:
We should define it outside of the function. Also, we can change it human-friendly.
```suggestion
	const (
		MiB = 1024*1024
		MaxFileSize = 200 * MiB
	)
```

Reply:
changed.